tla/consensus/ccfraft.tla

```diff
@@ -1274,11 +1260,14 @@ MatchIndexLowerBoundSentIndexInv ==
 
 CommitCommittableIndices ==
     \A i \in Servers :
-        committableIndices[i] # {} => commitIndex[i] < Min(committableIndices[i])
+        \/
+            /\ commitIndex[i] = 0
+            /\ CommittableIndices(i) = {}
+        \/ commitIndex[i] \in CommittableIndices(i)
 
 CommittableIndicesAreKnownSignaturesInv ==
     \A i \in Servers :
-        \A j \in committableIndices[i] :
+        \A j \in CommittableIndices(i) :
```

Comment:
This invariant could go entirely with the current formulation. If we derive `Committable` as we currently do, this is a tautology. It was added when we had explicit state, to try and confirm the state was sensible.

Reply:
Agreed